specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2019-05-13/bms.json

```diff
@@ -0,0 +1,3352 @@
+          "enum": [
+            "Passed",
+            "ActionRequired",
+            "ActionSuggested",
+            "Invalid"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "HealthStatus",
+            "modelAsString": true
+          }
+        },
+        "lastBackupStatus": {
+          "description": "Last backup operation status. Possible values: Healthy, Unhealthy.",
+          "type": "string"
+        },
+        "lastBackupTime": {
+          "format": "date-time",
+          "description": "Timestamp of the last backup operation on this backup item.",
+          "type": "string"
+        },
+        "extendedInfo": {
+          "$ref": "#/definitions/AzureFileshareProtectedItemExtendedInfo",
+          "description": "Additional information with this backup item."
+        }
+      },
+      "x-ms-discriminator-value": "AzureFileShareProtectedItem"
+    },
+    "AzureFileshareProtectedItemExtendedInfo": {
+      "description": "Additional information about Azure File Share backup item.",
+      "type": "object",
+      "properties": {
+        "oldestRecoveryPoint": {
+          "format": "date-time",
+          "description": "The oldest backup copy available for this item in the service.",
+          "type": "string"
+        },
+        "recoveryPointCount": {
+          "format": "int32",
+          "description": "Number of available backup copies associated with this backup item.",
+          "type": "integer"
+        },
+        "policyState": {
+          "description": "Indicates consistency of policy object and policy applied to this backup item.",
+          "type": "string"
+        },
+        "resourceState": {
+          "description": "Indicates the state of this resource. Possible values are from enum ResourceState {Invalid, Active, SoftDeleted, Deleted}",
+          "type": "string"
+        },
+        "resourceStateSyncTime": {
+          "format": "date-time",
+          "description": "The resource state sync time for this backup item.",
+          "type": "string"
+        }
+      }
+    },
+    "AzureFileShareRecoveryPoint": {
+      "description": "Azure File Share workload specific backup copy.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/RecoveryPoint"
+        }
+      ],
+      "properties": {
+        "recoveryPointType": {
+          "description": "Type of the backup copy. Specifies whether it is a crash consistent backup or app consistent.",
+          "type": "string"
+        },
+        "recoveryPointTime": {
+          "format": "date-time",
+          "description": "Time at which this backup copy was created.",
+          "type": "string"
+        },
+        "fileShareSnapshotUri": {
+          "description": "Contains Url to the snapshot of fileshare, if applicable",
+          "type": "string"
+        },
+        "recoveryPointSizeInGB": {
```

Comment:
Is this readonly? (or any other properties on this type definition)

Reply:
Yes